### PR TITLE
Prepare release of 6.5.3

### PR DESCRIPTION
**Pull request by Nyholm**
This release will include a fix for #2511

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 6.5.3 - 2020-04-17
+
+* Use Symfony  intl-idn polyfill [#2550](https://github.com/guzzle/guzzle/pull/2550)
```
Comment (GrahamCampbell) [inline CHANGELOG.md:5]: Too many spaces after `Symfony`. ;)

Comment (Nyholm) [reply CHANGELOG.md]: Thank you

**Comment by Nyholm** [comment]
oops. Thank you

**Comment by Nyholm** [comment]
Thank you. 
I'll merge and tag this.

**Comment by GrahamCampbell** [comment]
Can we do a 6.5 -> master branch merge and think about a 7.0 beta 2? Let me know if there's anything I can do to help. In particular, any such branch merge won't need to merge the global function changes, since a more drastic version was already applied to master. Just the idn stuff.

**Comment by Nyholm** [comment]
Yeah, Or is it easier to just make a new IDN PR to master?

**Comment by GrahamCampbell** [comment]
I'll take a look now.

**Comment by GrahamCampbell** [comment]
https://github.com/guzzle/guzzle/pull/2618 should do the job. :)

**Pull request merged**